src/work-allocation-2/utils/work-allocation-utils.ts

```diff
@@ -59,6 +59,29 @@ export const handleFatalErrors = (status: number, navigator: Navigator, fatals?:
   }
 };
 
+export const handleTasksFatalErrors = (status: number, navigator: Navigator, fatals?: FatalRedirect[]): number => {
```

Comment:
Minor comment - Is this handleTaskFatalErrors necessary as handleFatalErrors is already present in the file and there is one small change for 400 errors?

Reply:
At first glance it's not necessary however `handleFatalErrors` is a generic error function used in multiple places. Changing how we handle errors, just to cater for tasks screen will break the business rules on other screens.

Reply:
Agree with @connorpgpmcelroy  here, The generic error handling should be enough. Are you saying that this change is only for the Tasks Screen?

Reply:
@udaydenduluri33 yes. The solution is reuse `handleFatalErrors` function and add an extra parameter to treat 400 errors from that screen differently or create a new function, I chose the later.

Reply:
Ok makes sense